deliverables/common_problems.md

```diff
@@ -0,0 +1,12 @@
+# Common Problems and their Solutions
+
+This page collates some problems that were hit when transitioning to Python 3
+and their respective solutions.
+
+## Boost in a Python 3 Flavour of CY2019
+
+Unfortunately Python 3.7.x cannot be added to the CY2019 version
+of the VFX Reference Platform as-is.
+Boost 1.66 fails to find Python correctly when building with Python support.
+The easiest fix we (DNEG) found was to upgrade Boost to 1.70,
+as this is the version of Boost defined in CY2020.
```

Comment:
Maybe not CY2019 only but for a CMake build system, we (Animal Logic) found that Pixar's USD has a very useful module to find the correct Boost Python version:
https://github.com/PixarAnimationStudios/USD/blob/master/cmake/defaults/Packages.cmake#L42-L86

Reply:
Please could you write this up and add it to the document after this PR is merged.